packages/autotest/src/github/GitHubAutoTest.ts

```diff
@@ -180,7 +180,7 @@ export class GitHubAutoTest extends AutoTest implements IGitHubTestManager {
         if (delivId === null) {
```

Comment:
These tests likely did not have to change.

Taking out the feedback mode consistent with earlier logic. Looks like we need this.